kythe/typescript/indexer.ts

```diff
@@ -844,21 +898,28 @@ class Visitor {
 
```

Comment:
I thought you were moving this aliasing logic into the SymbolVNameCache. Are you still planning on that, or is this the final state?

Reply:
?